src/main/java/redis/clients/jedis/JedisSentinelPool.java

```diff
@@ -84,14 +87,26 @@ public JedisSentinelPool(String masterName, Set<String> sentinels,
   }
 
   public JedisSentinelPool(String masterName, Set<String> sentinels,
-      final GenericObjectPoolConfig poolConfig, final int connectionTimeout, final int soTimeout,
-      final String password, final int database, final String clientName) {
+      final GenericObjectPoolConfig poolConfig, final int connectionTimeout, final int soTimeout, final String password,
+	   final int database, final String clientName) {
+    this(masterName, sentinels, poolConfig, connectionTimeout, soTimeout, password, database, clientName,
+        Protocol.DEFAULT_TIMEOUT, Protocol.DEFAULT_TIMEOUT, null);
+  }
+
+  public JedisSentinelPool(String masterName, Set<String> sentinels,
+      final GenericObjectPoolConfig poolConfig, final int connectionTimeout, final int soTimeout, final String password,
+       final int database, final String clientName,
+       final int sentinelConnectionTimeout, final int sentinelSoTimeout, final String sentinelPassword) {
```

Comment:
I think, after this, a request for sentinelClientName would come right away. IMO, we can add it now.

Reply:
Agreed. I'll add an alternative constructor, similar to initialising the pool with/without a client name. Unless you disagree?

Reply:
Let's just do it in one constructor.